I/IGLWrap/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+#=
+BINARYBUILDER_RUNNER=privileged julia build_tarballs.jl --verbose --debug       
+=#
+
+using BinaryBuilder
+using Pkg
+
+version = v"2.3.0" # libIGL version, see below:
+
+sources = [
+	DirectorySource("./iglwrap"),
+	# 2.3.0 stable release:
+	GitSource("https://github.com/libigl/libigl.git",
+		"e60423e28c86b6aa2a3f6eb0112e8fd881f96777")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p build
+cd build
+# this line seems to prevent compilation; removed for testing:
+# -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_MODULE_PATH=${WORKSPACE}/srcdir/libigl/cmake\; \
+    ..
+make -j${nproc}
+make install
+
+# install_license ${WORKSPACE}/srcdir/LICENSE
```

Comment:
The license is still missing

Reply:
Since (almost) no other Yggdrasil package uses a LICENSE file , I figured that copying LibIGL's MPL license file was useless here as well. That file is removed for good, unless I need to somehow put a license on the C++ -> C wrapper.

Reply:
> Since (almost) no other Yggdrasil package uses a LICENSE file

All packages should have a license file.  In most cases that's automatically installed, but not here, so you have to manually add the step to install it: https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file